[SDCT-509] Document @test.fingerprint_fqn facet for Test Optimization (#37886)
DataDog/documentation: content/en/tests/explorer/_index.md

| Standard Deviation Change | Indicates if the test has been newly added. |
| Test Code Owners | The name of the test's codeowners as inferred from the repository configuration. |
| Test Fingerprint | The unique identifier for an individual test run. |

> | Test Fingerprint (FQN) | A stable, unique identifier for a test (`@test.fingerprint_fqn`), derived from a hash of the repository ID and the test's fully qualified name. Because it stays the same across runs, use it to consistently identify the same test — for example, the test ID used by the [Flaky Tests Management API][19]. |

Reviewer: ```suggestion
| Test Fingerprint (FQN) | A stable, unique identifier for a test (`@test.fingerprint_fqn`) across all of its runs, derived from a hash of the repository ID and the test's fully qualified name (FQN). Use it to consistently identify the same test over time. This is also the identifier used by the [Flaky Tests Management API][19]. |
```
